ec2-cluster/modules/node/templates/l1-initial-validator

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+export CL_KEYSTORE=../key.p12 && \
```

Comment:
Even if L1 is using the same key as L0, please don't refer to the same key, just make a copy of it (or assume that it's copied there).